VAMobile/src/components/CommonErrorComponents/CustomError.tsx

```diff
@@ -60,7 +60,10 @@ const CustomError: FC<CustomErrorProps> = ({ onTryAgain, titleText, errorText, c
   return (
     <VAScrollView contentContainerStyle={scrollStyles}>
       <Box justifyContent="center" {...containerStyles}>
-        <AlertBox title={fixSpacingAndSpecialCharacters(titleText)} titleA11yLabel={titleA11y} border="error">
+        <AlertWithHaptics
+          variant="error"
+          header={fixSpacingAndSpecialCharacters(titleText)}
+          headerA11yLabel={titleA11y}>
           <Box>
```

Comment:
Could potentially be further simplified to using Alert component description/primaryButton props leaving only the ClickToCallPhoneNumber as children.